### ISSUE TITLE
C++ attributes are not shown in gray color in the editor if they span multiple lines

### ISSUE THREAD

**Issue by VladimirReshetnikov** [compiler-explorer/compiler-explorer#878]
**ACTUAL:**
```
[[using gnu: 
   unused]] 
int x;
```

![image](https://user-images.githubusercontent.com/1231606/37792613-72e2646a-2dc9-11e8-9e89-623c655c5d01.png)

**EXPECTED:**
I would expect that multi-line attributes are shown with the same style as single-line ones, like this:

```
[[using gnu: unused]] 
int x;
```

![image](https://user-images.githubusercontent.com/1231606/37792691-af387aee-2dc9-11e8-82e3-5f528226b400.png)

https://godbolt.org/z/drzEYY

**Comment by RabsRincon** [comment]
Hi! Thank you for taking the time to report the three (with #879 & #880) issues with the highlighter. 
Note that while we can sometimes fix stuff like this (we already do a bit of work in the highlighter) this is in fact a problem with the editor we use: [Monaco editor](https://github.com/Microsoft/monaco-editor).

I encourage you to report these issues directly to them. That way there's a greater chance of them being fixed, and then we would not be the only ones benefitting from it :)

Please let us know when you do so, so we can track when they're fixed too.

**Comment by RabsRincon** [comment]
I forgot to mention that even tho it's a Monaco flaw, I'll fix this locally as soon as I get some free time :)

**Comment by mattgodbolt** [comment]
This isn't fixed in monaco 18.1 either, just fyi (as I happened to be checking)

**Comment by mattgodbolt** [comment]
@RubenRBS sorry to steal your thunder here - I went straight to the source :)
